cpp/CMakeLists.txt

```diff
@@ -18,6 +18,37 @@
 cmake_minimum_required(VERSION 3.2)
 message(STATUS "Building using CMake version: ${CMAKE_VERSION}")
 
+cmake_policy(SET CMP0025 NEW)
+# IN_LIST if() operator
+cmake_policy(SET CMP0057 NEW)
+
+# Compatibility with CMake 3.1
+if(POLICY CMP0054)
+  # http://www.cmake.org/cmake/help/v3.1/policy/CMP0054.html
```

Comment:
Could you also add the one-line summary in the page ("Only interpret if() arguments as variables or keywords when unquoted.") to here?
It's difficult to know what is the policy for only with URL...